sherlock/resources/data.json

```diff
@@ -651,6 +651,14 @@
     "username_claimed": "blue",
     "username_unclaimed": "noonewouldeverusethis7"
   },
+  "GitHub Support Community": {
+    "errorMsg": "Oops! That page doesn’t exist or is private.",
+    "errorType": "message",
+    "url": "https://github.community/u/{}/summary",
+    "urlMain": "https://github.community",
+    "username_claimed": "jperl",
+    "username_unclaimed": "blue"
```

Comment:
For the first time that I've seen, "blue" is unclaimed.

Reply:
Instead of using `blue` as the claimed username, use something more obscure such as `noonewouldusethis298` because there is a very high chance of the username `blue` being claimed in the future on GitHub Support Communites